[DOCS-1484] Document username run filter for projects (#1650)
wandb/docs: content/en/guides/core/automations/automation-events.md

Learn more about [creating automations]({{< relref "create-automations/" >}}).

### Scopes
A [Registry]({{< relref "/guides/core/registry/">}}) automation watches for the event taking place on any collection within a specific registry, including collections added in the future.

> ### Events {#registry-events}

Reviewer: Is this syntax Hugo specific? If yay, should we omit since it'll get lost in the Mintlify migration?

Author: It's not really Hugo-specific but it doesn't seem to be supported in Mintlify. But Mintlify supports putting `<a>` anchors in the header to override the anchor. I wanted to name the anchor here to avoid having `#events` and `#events-1`, which is what Hugo does. In Jekyll, the second anchor would just not be reachable, so I tend to try to avoid duplicates.

I'll leave this for now because we will catch it when catching the content up in Mintlify.

Learn more about [creating automations]({{< relref "create-automations/" >}}).

### Scopes
A [Registry]({{< relref "/guides/core/registry/">}}) automation watches for the event taking place on any collection within a specific registry, including collections added in the future.

> ### Events

Author: ```suggestion
### Events {#registry-events}
```

Learn more about [creating automations]({{< relref "create-automations/" >}}).

### Scopes
A project-level automation watches for the event taking place on any collection in the project. Depending on the event you specify, you can further limit the scope of the automation.

### Artifact events
This section describes the events related to an artifact that can trigger an automation.

> - **A new version is added to an artifact**: Apply recurring actions to each version of an artifact. For example, start a training job when a new dataset artifact version is created. To limit the automation's scope, in **Artifact filter**, select a specific artifact.

Reviewer: ```suggestion
- **A new version is added to an artifact**: Apply recurring actions to each version of an artifact. For example, start a training job when a new dataset artifact version is created. To limit the automation's scope, select a specific artifact in the **Artifact filter** field.
```

#### Run filters
This section describes how the automation selects runs to evaluate.

> - By default, any run in the project triggers the automation when the event occurs. To consider only specific runs, specify one or more run filters:

Author: This intentionally doesn't use the include because it's laid out a bit differently

> - By default, any run in the project triggers the automation when the event occurs. To consider only specific runs, specify one or more run filters:

Reviewer: "To consider only specific runs..." sounds a little confusing. Let's make it more simple, something like:

```suggestion
- By default, any run in the project triggers the automation when the event occurs. You can limit which runs trigger an automation by configuring one of the following filters: 
```
